README.md

```diff
@@ -32,14 +32,26 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. What is the DOM?
 
+The DOM is the document object model. It is an API, taking all of our code and displaying it in the browser starting with the HTML first usually. 
+
 2. What is an event?
 
+An event is an action that will cause a javascript code to excecute. 
```

Comment:
It will cause code to execute when used with an event listener. This is a hard one to put into word. Here was my answer for the same question:
`
An event is something that happens... Usually we think of events as something that's triggered by a user, but it can also be generated by APIs. Each event has a type, which inherits various attributes and methods.
`